.github/workflows/sdist.yml

```diff
@@ -14,78 +14,87 @@ on:
       - "doc/**"
 
 jobs:
-  build:
-    if: ${{ github.event.label.name == 'Build' || contains(github.event.pull_request.labels.*.name, 'Build') || github.event_name == 'push'}}
+  sdist:
+    #if: ${{ github.event.label.name == 'Build' || contains(github.event.pull_request.labels.*.name, 'Build') || github.event_name == 'push'}}
```

Comment:
Why was this commented? sdists are not supposed to run on every PR as per #43745

Reply:
Thanks, that's a debugging leftover